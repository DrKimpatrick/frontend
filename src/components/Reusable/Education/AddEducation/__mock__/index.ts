export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  messages: {
    message: 'sent'
  },
  educations: {
    loading: false,
    errors: {
      errors: null
    },
    education: {
      _id: 'id',
      schoolName: 'school name',
      level: 'level',
      degreeOrCertification: 'education',
      specializations: ['specialization'],
      startDate: '2020-10-10',
      endDate: '2020-10-5',
      accomplishments: ['snow'],
      isCurrentEducation: false,
      verificationStatus: 'verified'
    }
  }
};
