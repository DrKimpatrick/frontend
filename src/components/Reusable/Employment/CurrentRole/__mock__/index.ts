export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow'
    },
    errors: null,
    loading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  }
};
