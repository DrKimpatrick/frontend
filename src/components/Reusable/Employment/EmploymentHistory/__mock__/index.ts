export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow',
      supervisor: 'supervisor',
      title: 'title',
      startDate: '2020-10-5',
      responsibilities: ['res'],
      accomplishments: ['res'],
      favoriteProject: 'res',
      skillsUsed: []
    },
    errors: null,
    loading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  },
  skills: {
    userSkill: [
      {
        supervisor: {
          detail: {
            name: 'user',
            email: 'email@gmail.com',
            phoneNumber: '0786601003'
          },
          name: 'HR'
        },
        skillsUsed: [],
        responsibilities: [],
        accomplishments: [],
        verificationStatus: 'unverified',
        isCurrentPosition: true,
        _id: '5fcf65a2c4c797a20acc1c99',
        title: 'Product Manager',
        startDate: '2020-10-15T00:00:00.000Z',
        endDate: null,
        companyName: 'heza',
        favoriteProject: 'tembea',
        userId: '5fcdf47ea1d1dd1564a21613',
        createdAt: '2020-12-08T11:38:10.729Z',
        updatedAt: '2020-12-08T11:38:10.729Z'
      }
    ]
  }
};
