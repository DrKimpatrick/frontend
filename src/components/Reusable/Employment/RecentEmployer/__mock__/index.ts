export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {
      _id: 'id',
      username: 'username',
      email: 'email',
      roles: ['talent'],
      updatedAt: '2020-10-10',
      dateRegistered: '2020-10-10',
      paymentStatus: 'verified',
      featureChoice: 'local',
      verified: true
    }
  },
  employments: {
    employment: {
      companyName: 'snow',
      supervisor: 'supervisor',
      title: 'title',
      startDate: '2020-10-5',
      responsibilities: ['res'],
      accomplishments: ['res'],
      favoriteProject: 'res',
      _id: 'id'
    },
    errors: null,
    loading: false,
    submitLoading: false
  },
  messages: {
    message: 'sent'
  },
  educations: {
    education: {
      schoolName: 'schoolName'
    }
  },
  skills: {
    userSkill: [
      {
        skill: {
          _id: '5fcf1c5b11c16c41f7c6c723',
          skill: 'Python',
          createdAt: '2020-12-08T06:25:31.128Z',
          updatedAt: '2020-12-08T06:25:31.128Z'
        },
        level: 'intermediate',
        verificationStatus: 'verified',
        _id: '5fcf52105cd56b9843252696',
        createdAt: '2020-12-08T10:14:40.231Z',
        updatedAt: '2020-12-08T13:05:46.873Z'
      }
    ]
  }
};
