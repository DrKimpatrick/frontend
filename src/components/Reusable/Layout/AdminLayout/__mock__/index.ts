export const initialState = {
  users: {
    currentUser: {
      isLoggedIn: true,
      loading: false,
      error: null,
      message: null,
      created: false,
      data: {
        firstName: 'John',
        lastName: 'Smith',
        profilePicture:
          'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png'
      }
    },
    user: {}
  }
};
