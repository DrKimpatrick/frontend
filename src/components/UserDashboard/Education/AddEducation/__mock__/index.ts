export const initialState = {
  educations: {
    loading: false,
    errors: {
      errors: null
    }
  },
  messages: {
    message: 'message'
  }
};
