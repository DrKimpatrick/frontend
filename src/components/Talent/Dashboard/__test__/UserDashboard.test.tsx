import { cleanup } from '@testing-library/react';
import React from 'react';
import ReactDom from 'react-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import { Dashboard } from '..';
import { initialState } from '../__mock__';
import { TalentProcess } from 'redux/action-types/user';

let store: any;

const middleware = [thunk];

const mockStore = configureMockStore(middleware);

describe('User Dashboard', () => {
  beforeEach(() => {
    store = mockStore(initialState);
  });

  afterEach(cleanup);

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDom.render(
      <Provider store={store}>
        <Router>
          <Dashboard />
        </Router>
      </Provider>,
      div
    );
  });

  it('should create a snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Router>
            <Dashboard />
          </Router>
        </Provider>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  describe('Profile Process', () => {
    beforeEach(() => {
      store = mockStore({
        ...initialState,
        users: {
          ...initialState.users,
          user: {
            ...initialState.users.user,
            profileProcess: TalentProcess.AddEducation
          }
        }
      });
    });

    it('should redirect user back', () => {
      const tree = renderer
        .create(
          <Provider store={store}>
            <Router>
              <Dashboard />
            </Router>
          </Provider>
        )
        .toJSON();

      expect(tree).toBeDefined();
    });
  });
});
