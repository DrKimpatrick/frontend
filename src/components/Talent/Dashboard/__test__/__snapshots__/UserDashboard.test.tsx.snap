// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Dashboard should create a snapshot 1`] = `
Array [
  <div
    className="flex flex-col min-w-0 bg-clip-border relative cover-image w-full"
  >
    <img
      alt=""
      className="uploaded-image"
      src="https://www.qualitylogic.com/wp-content/uploads/2018/09/learning.svg"
    />
    <div
      className="absolute inset-0 coverImageLogo"
    >
      <img
        alt=""
        className="bg-white w-16 h-16 rounded-full object-contain my-8  mx-12 shadow"
        src="logo-image.png"
      />
      <div
        className="w-full text-center edit-upload-icon"
      >
        <button
          className="outline-none border-none m-auto px-2 py-2 bg-transparent border-solid border-gray-500 shadow-lg rounded-sm text-center text-white"
          type="submit"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34a.9959.9959 0 00-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z"
            />
          </svg>
           Edit cover image
        </button>
      </div>
    </div>
  </div>,
  <nav
    className="flex items-center bg-white-800 px-20 false flex-wrap"
    style={
      Object {
        "background": "white",
      }
    }
  >
    <div
      className="mt-24 mb w-3/5 flex-nowrap"
    >
      <ul
        className="flex flex-nowrap justify-between dashboard-items-ul"
      >
        <li
          className=""
        >
          <a
            href="/user/dashboard"
            onClick={[Function]}
          >
            <div
              className="flex flex-nowrap text-sm text-gray-600"
            >
              <span
                className="px-2 text-xs"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root dashboard-list-icons"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M3 13h8V3H3v10zm0 8h8v-6H3v6zm10 0h8V11h-8v10zm0-18v6h8V3h-8z"
                  />
                </svg>
              </span>
               
              <span
                className="px-2"
              >
                Dashboard
              </span>
            </div>
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/"
            onClick={[Function]}
          >
            <div
              className="flex flex-nowrap text-sm text-gray-600"
            >
              <span
                className="px-2 text-xs"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root dashboard-list-icons"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 6h-4V4c0-1.11-.89-2-2-2h-4c-1.11 0-2 .89-2 2v2H4c-1.11 0-1.99.89-1.99 2L2 19c0 1.11.89 2 2 2h16c1.11 0 2-.89 2-2V8c0-1.11-.89-2-2-2zm-6 0h-4V4h4v2z"
                  />
                </svg>
              </span>
               
              <span
                className="px-2"
              >
                Applied Jobs
              </span>
            </div>
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/training"
            onClick={[Function]}
          >
            <div
              className="flex flex-nowrap text-sm text-gray-600"
            >
              <span
                className="px-2 text-xs"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root dashboard-list-icons"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 6h-4V4c0-1.11-.89-2-2-2h-4c-1.11 0-2 .89-2 2v2H4c-1.11 0-1.99.89-1.99 2L2 19c0 1.11.89 2 2 2h16c1.11 0 2-.89 2-2V8c0-1.11-.89-2-2-2zm-6 0h-4V4h4v2z"
                  />
                </svg>
              </span>
               
              <span
                className="px-2"
              >
                Trainings
              </span>
            </div>
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="/t/recommendation"
            onClick={[Function]}
          >
            <div
              className="flex flex-nowrap text-sm text-gray-600"
            >
              <span
                className="px-2 text-xs"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root dashboard-list-icons"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 6h-4V4c0-1.11-.89-2-2-2h-4c-1.11 0-2 .89-2 2v2H4c-1.11 0-1.99.89-1.99 2L2 19c0 1.11.89 2 2 2h16c1.11 0 2-.89 2-2V8c0-1.11-.89-2-2-2zm-6 0h-4V4h4v2z"
                  />
                </svg>
              </span>
               
              <span
                className="px-2"
              >
                Recommendation
              </span>
            </div>
          </a>
        </li>
      </ul>
    </div>
    <div
      className="hidden top-navbar w-full lg:inline-flex lg:flex-grow lg:w-auto"
      id="navigation"
    >
      <div
        className="lg:inline-flex lg:flex-row lg:ml-auto lg:w-auto w-full lg:items-center items-start flex flex-col lg:h-auto"
      >
        <div
          className="lg:inline-flex items-center -mt-24"
        >
          <div
            className="px-8 lg:inline-flex items-center"
          >
            <div
              className="makeStyles-root-1"
            >
              <div>
                <div
                  aria-haspopup="true"
                  className="notification-container"
                  onClick={[Function]}
                  onTouchEnd={[Function]}
                >
                  <div
                    className="text-gray-600 relative notification-icon"
                    onClick={[Function]}
                  >
                    <div
                      className="bg-white text-red-600 h-3 w-3 rounded-full absolute right-0 top-0 flex items-center justify-center border-red-600 border border-solid"
                      style={
                        Object {
                          "fontSize": "6px",
                        }
                      }
                    >
                      <span>
                        9+
                      </span>
                    </div>
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root cursor-pointer"
                      focusable="false"
                      style={
                        Object {
                          "color": "#ff0000",
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 22c1.1 0 2-.9 2-2h-4c0 1.1.89 2 2 2zm6-6v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-.83-.67-1.5-1.5-1.5s-1.5.67-1.5 1.5v.68C7.63 5.36 6 7.92 6 11v5l-2 2v1h16v-1l-2-2z"
                      />
                    </svg>
                  </div>
                  <div
                    className="header-popup tri-right border right-top hide"
                  >
                    <div
                      className="text-gray-600 w-56 space-x-2 text-sm py-2 flex "
                    >
                      <span
                        className="text-red-600 font-bold"
                      >
                        1
                      </span>
                      <div
                        className="flex flex-col w-full flex-wrap"
                      >
                        <span
                          className="text-red-600 font-bold"
                        >
                          You have been verified
                        </span>
                        <p
                          className="break-normal"
                        >
                          Check your email for the link verification
                        </p>
                      </div>
                    </div>
                    <div
                      className="text-gray-600 w-56 space-x-2 text-sm py-2 flex "
                    >
                      <span
                        className="text-red-600 font-bold"
                      >
                        1
                      </span>
                      <div
                        className="flex flex-col w-full flex-wrap"
                      >
                        <span
                          className="text-red-600 font-bold"
                        >
                          You have been verified
                        </span>
                        <p
                          className="break-normal"
                        >
                          Check your email for the link verification
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="text-gray-600 relative"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11.99 18.54l-7.37-5.73L3 14.07l9 7 9-7-1.63-1.27-7.38 5.74zM12 16l7.36-5.73L21 9l-9-7-9 7 1.63 1.27L12 16z"
                />
              </svg>
            </div>
          </div>
          <div
            className="text-gray-600"
          >
            <span>
              username
            </span>
          </div>
          <div
            className="makeStyles-root-1"
          >
            <div>
              <div
                aria-haspopup="true"
                className="profile-container"
                onClick={[Function]}
                onTouchEnd={[Function]}
              >
                <div
                  aria-haspopup="true"
                  onClick={[Function]}
                >
                  <img
                    alt="avatar"
                    className="inline-block ml-2 h-24 w-24 rounded-full text-white shadow-solid cursor-pointer"
                    src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
                  />
                </div>
                <div
                  className="header-popup tri-left border right-top hide"
                >
                  <ul
                    className="MuiList-root MuiList-padding"
                    onKeyDown={[Function]}
                    role="menu"
                    tabIndex={-1}
                  >
                    <li
                      aria-disabled={false}
                      className="MuiButtonBase-root MuiListItem-root MuiMenuItem-root text-gray-600 MuiMenuItem-gutters MuiListItem-gutters MuiListItem-button"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      role="menuitem"
                      tabIndex={0}
                    >
                      <div
                        className="text-gray-600 w-56 space-x-3 text-sm py-2"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M3 13h8V3H3v10zm0 8h8v-6H3v6zm10 0h8V11h-8v10zm0-18v6h8V3h-8z"
                          />
                        </svg>
                        <span>
                          Dashboard
                        </span>
                      </div>
                      <span
                        className="MuiTouchRipple-root"
                      />
                    </li>
                    <li
                      aria-disabled={false}
                      className="MuiButtonBase-root MuiListItem-root MuiMenuItem-root text-gray-600 MuiMenuItem-gutters MuiListItem-gutters MuiListItem-button"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      role="menuitem"
                      tabIndex={-1}
                    >
                      <div
                        className="text-gray-600 w-56 space-x-3 text-sm py-2"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M19.14 12.94c.04-.3.06-.61.06-.94 0-.32-.02-.64-.07-.94l2.03-1.58c.18-.14.23-.41.12-.61l-1.92-3.32c-.12-.22-.37-.29-.59-.22l-2.39.96c-.5-.38-1.03-.7-1.62-.94l-.36-2.54c-.04-.24-.24-.41-.48-.41h-3.84c-.24 0-.43.17-.47.41l-.36 2.54c-.59.24-1.13.57-1.62.94l-2.39-.96c-.22-.08-.47 0-.59.22L2.74 8.87c-.12.21-.08.47.12.61l2.03 1.58c-.05.3-.09.63-.09.94s.02.64.07.94l-2.03 1.58c-.18.14-.23.41-.12.61l1.92 3.32c.12.22.37.29.59.22l2.39-.96c.5.38 1.03.7 1.62.94l.36 2.54c.05.24.24.41.48.41h3.84c.24 0 .44-.17.47-.41l.36-2.54c.59-.24 1.13-.56 1.62-.94l2.39.96c.22.08.47 0 .59-.22l1.92-3.32c.12-.22.07-.47-.12-.61l-2.01-1.58zM12 15.6c-1.98 0-3.6-1.62-3.6-3.6s1.62-3.6 3.6-3.6 3.6 1.62 3.6 3.6-1.62 3.6-3.6 3.6z"
                          />
                        </svg>
                        <span>
                          Settings
                        </span>
                      </div>
                      <span
                        className="MuiTouchRipple-root"
                      />
                    </li>
                    <li
                      aria-disabled={false}
                      className="MuiButtonBase-root MuiListItem-root MuiMenuItem-root text-gray-600 MuiMenuItem-gutters MuiListItem-gutters MuiListItem-button"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      role="menuitem"
                      tabIndex={-1}
                    >
                      <div
                        className="text-gray-600 w-56 space-x-3 text-sm py-2"
                        onClick={[Function]}
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M13 3h-2v10h2V3zm4.83 2.17l-1.42 1.42C17.99 7.86 19 9.81 19 12c0 3.87-3.13 7-7 7s-7-3.13-7-7c0-2.19 1.01-4.14 2.58-5.42L6.17 5.17C4.23 6.82 3 9.26 3 12c0 4.97 4.03 9 9 9s9-4.03 9-9c0-2.74-1.23-5.18-3.17-6.83z"
                          />
                        </svg>
                        <span>
                          Logout
                        </span>
                      </div>
                      <span
                        className="MuiTouchRipple-root"
                      />
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </nav>,
  <div
    className="user-dashboard-container mx-auto"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <div
        className="w-full verification"
      >
        <ul
          className="flex justify-between mx-auto"
        >
          <li
            className="pb-4"
          >
            <div
              className="flex flex-row text-sm"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root verified-color"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 1L3 5v6c0 5.55 3.84 10.74 9 12 5.16-1.26 9-6.45 9-12V5l-9-4zm-2 16l-4-4 1.41-1.41L10 14.17l6.59-6.59L18 9l-8 8z"
                />
              </svg>
              <h4
                className="verified-color px-4"
              >
                Certified and Verified
              </h4>
            </div>
          </li>
          <li
            className="pb-4"
          >
            <div
              className="flex flex-row text-sm"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root verify-progress-color"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M13 2.05v3.03c3.39.49 6 3.39 6 6.92 0 .9-.18 1.75-.48 2.54l2.6 1.53c.56-1.24.88-2.62.88-4.07 0-5.18-3.95-9.45-9-9.95zM12 19c-3.87 0-7-3.13-7-7 0-3.53 2.61-6.43 6-6.92V2.05c-5.06.5-9 4.76-9 9.95 0 5.52 4.47 10 9.99 10 3.31 0 6.24-1.61 8.06-4.09l-2.6-1.53C16.17 17.98 14.21 19 12 19z"
                />
              </svg>
              <h4
                className="verify-progress-color px-4"
              >
                Verification in progress
              </h4>
            </div>
          </li>
          <li
            className="pb-4"
          >
            <div
              className="flex flex-row text-sm"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root not-verified-color"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 2H4c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-7 9h-2V5h2v6zm0 4h-2v-2h2v2z"
                />
              </svg>
              <h4
                className="not-verified-color px-4"
              >
                Not Verified
              </h4>
            </div>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="pb-48 tDashboardItem"
    >
      <div
        className="MuiContainer-root MuiContainer-maxWidthLg"
      >
        <div
          className="flex flex-nowrap w-full justify-between"
        >
          <div
            className="flex flex-col w-2/5 mt-4"
          >
            <div
              className="mb-16"
            >
              <div
                className="flex flex-no-wrap uppercase font-thin items-center"
              >
                <div
                  className="light-fonts"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M21 2H3c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h7v2H8v2h8v-2h-2v-2h7c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm0 14H3V4h18v12z"
                    />
                  </svg>
                </div>
                <span
                  className="px-2"
                >
                  Employment
                </span>
              </div>
              <div
                className="w-full"
              >
                <p
                  className="text-right text-xs py-1 pb-2"
                />
                <ul
                  className="p-0 bg-card-preview"
                >
                  <li
                    className="py-2 py-2 rounded-sm action"
                    onClick={[Function]}
                    onMouseLeave={[Function]}
                    onMouseOver={[Function]}
                  >
                    <div
                      className="flex flex-row text-sm justify-between"
                    >
                      <h4
                        className="medium-fonts px-4"
                      >
                        snow
                      </h4>
                      <div
                        className="flex flex-row"
                      />
                    </div>
                  </li>
                </ul>
                <div
                  className="addButton"
                >
                  <button
                    className="mt-4 bg-gray-800 w-full text-white hover:bg-gray-900 outline-none font-semibold h-12 py-1 px-1 rounded-sm shadow addEmployment"
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                      />
                    </svg>
                     
                    <span>
                      Add new Employment
                    </span>
                  </button>
                </div>
              </div>
            </div>
            <div>
              <div
                className="flex flex-no-wrap uppercase font-thin items-center"
              >
                <div
                  className="light-fonts"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M5 13.18v4L12 21l7-3.82v-4L12 17l-7-3.82zM12 3L1 9l11 6 9-4.91V17h2V9L12 3z"
                    />
                  </svg>
                </div>
                <span
                  className="px-2"
                >
                  Education
                </span>
              </div>
              <div
                className="w-full"
              >
                <p
                  className="text-right text-xs py-1 pb-2"
                />
                <ul
                  className="p-0 bg-card-preview"
                >
                  <li
                    className="py-2 py-2 rounded-sm action"
                    onClick={[Function]}
                    onMouseLeave={[Function]}
                    onMouseOver={[Function]}
                  >
                    <div
                      className="flex flex-row text-sm justify-between"
                    >
                      <h4
                        className="medium-fonts px-4"
                      >
                        schoolName
                      </h4>
                      <div
                        className="flex flex-row"
                      />
                    </div>
                  </li>
                </ul>
                <div
                  className="addButton"
                >
                  <button
                    className="mt-4 bg-gray-800 w-full text-white hover:bg-gray-900 outline-none font-semibold h-12 py-1 px-1 rounded-sm shadow addEducation"
                    onClick={[Function]}
                    type="button"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                      />
                    </svg>
                     
                    <span>
                      Add new Education
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
          <div
            className="flex flex-col w-2/5 mt-4"
          >
            <div>
              <div
                className="flex flex-no-wrap uppercase font-thin items-center"
              >
                <div
                  className="light-fonts"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20 6h-8l-2-2H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V8c0-1.1-.9-2-2-2zm-5 3c1.1 0 2 .9 2 2s-.9 2-2 2-2-.9-2-2 .9-2 2-2zm4 8h-8v-1c0-1.33 2.67-2 4-2s4 .67 4 2v1z"
                    />
                  </svg>
                </div>
                <span
                  className="px-2"
                >
                  SkillSet
                </span>
              </div>
              <div
                className="w-full"
              >
                <p
                  className="text-right text-xs py-2 pb-2 pr-3"
                >
                  Beginner
                </p>
                <ul
                  className="p-0 m-0 bg-card-preview "
                >
                  <li
                    className=" py-2 py-2 rounded-sm"
                    onClick={[Function]}
                    onMouseLeave={[Function]}
                    onMouseOver={[Function]}
                  >
                    <div
                      className="flex flex-row text-sm justify-between"
                    >
                      <h4
                        className="medium-fonts px-4"
                      >
                        php
                      </h4>
                      <div
                        className="flex flex-row"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root not-verified-color mr-4"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M20 2H4c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-7 9h-2V5h2v6zm0 4h-2v-2h2v2z"
                          />
                        </svg>
                      </div>
                    </div>
                  </li>
                </ul>
              </div>
              <div
                className="w-full"
              >
                <p
                  className="text-right text-xs py-2 pb-2 pr-3"
                >
                  Intermediate
                </p>
                <ul
                  className="p-0 m-0 bg-card-preview "
                >
                  <li
                    className=" py-2 py-2 rounded-sm"
                    onClick={[Function]}
                    onMouseLeave={[Function]}
                    onMouseOver={[Function]}
                  >
                    <div
                      className="flex flex-row text-sm justify-between"
                    >
                      <h4
                        className="medium-fonts px-4"
                      >
                        typescript
                      </h4>
                      <div
                        className="flex flex-row"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root not-verified-color mr-4"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M20 2H4c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-7 9h-2V5h2v6zm0 4h-2v-2h2v2z"
                          />
                        </svg>
                      </div>
                    </div>
                  </li>
                </ul>
              </div>
              <div
                className="w-full"
              >
                <p
                  className="text-right text-xs py-2 pb-2 pr-3"
                >
                  Advanced
                </p>
                <ul
                  className="p-0 m-0 bg-card-preview "
                >
                  <li
                    className=" py-2 py-2 rounded-sm"
                    onClick={[Function]}
                    onMouseLeave={[Function]}
                    onMouseOver={[Function]}
                  >
                    <div
                      className="flex flex-row text-sm justify-between"
                    >
                      <h4
                        className="medium-fonts px-4"
                      >
                        typescript
                      </h4>
                      <div
                        className="flex flex-row"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root not-verified-color mr-4"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M20 2H4c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-7 9h-2V5h2v6zm0 4h-2v-2h2v2z"
                          />
                        </svg>
                      </div>
                    </div>
                  </li>
                </ul>
              </div>
              <div
                className="addButton"
              >
                <button
                  className="mt-4 bg-gray-800 w-full text-white hover:bg-gray-900 outline-none font-semibold h-12 py-1 px-1 rounded-sm shadow add"
                  onClick={[Function]}
                  type="button"
                >
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                    />
                  </svg>
                   
                  <span>
                    Add new 
                  </span>
                   
                  <span>
                    skills
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="mainBackground"
  >
    <img
      alt=""
      src="motif-image.png"
      style={
        Object {
          "backgroundColor": "#c4bdbd",
        }
      }
    />
  </div>,
]
`;
